addons/mail/models/discuss/ir_websocket.py

```diff
@@ -61,6 +49,7 @@ def _build_presence_channel_list(self, presences):
     @add_guest_to_context
     def _build_bus_channel_list(self, channels):
         channels = list(channels)  # do not alter original list
+        presence_channels = self._build_presence_channel_list(channels)
         discuss_channel_ids = list()
```

Comment:
```suggestion
        discuss_channel_ids = []
```
:pick: 
![image](https://github.com/user-attachments/assets/ffce4bec-d595-4717-8709-fd3ce528255b)

faster and easier to read